### ISSUE TITLE
Bundesanzeiger: Only 1 item is returned for reports w/ same name

### ISSUE THREAD

**Issue by sklinkert** [bundesAPI/deutschland#82]
Some reports share the same names, e.g. `Mitteilung von Netto-Leerverkaufspositionen`. The problem is that `get_reports` cannot return multiple items with the same name because it returns a dictionary/map.

```python
#!/opt/homebrew/bin/python3
from deutschland.bundesanzeiger import Bundesanzeiger
ba = Bundesanzeiger()
data = ba.get_reports("DE000A0TGJ55")
print(data.keys())

print(data["Mitteilung von Netto-Leerverkaufspositionen"])
```

```
dict_keys(['Mitteilung von Netto-Leerverkaufspositionen'])
{'date': datetime.datetime(2022, 9, 26, 0, 0), 'name': 'Mitteilung von Netto-Leerverkaufspositionen', 'company': 'BlackRock Investment Management (UK) Limited', 'report': '\n\n\n\n\xa0\n\n\n\n\n\n\n\nBlackRock Investment Management (UK) Limited\nLondon\nMitteilung von Netto-Leerverkaufspositionen\nZu folgendem Emittenten wird vom oben genannten Positionsinhaber eine Netto-Leerverkaufsposition\n            gehalten:\n\nVARTA AKTIENGESELLSCHAFT\n\n\nISIN: DE000A0TGJ55\n\nDatum der Position: 23.09.2022\nProzentsatz des ausgegebenen Aktienkapitals: 2,26 %\n\xa0\n\n\n\n\n\n\n\n\n\n\n\n\n'}
```

Here you can see how it looks on the website:
<img width="1048" alt="Screenshot 2022-10-29 at 17 26 07" src="https://user-images.githubusercontent.com/4770842/198839846-aef7d283-9808-4693-8c76-ba2797497142.png">

**Comment by wirthual** [comment]
Thanks for reporting this.

Currently, the name is used to create the keys for the returned dictionary.

I created a Pull Request here, to generate a hash of the report to identify it. This will avoid clashing of the keys and should result in each entry listed in the results.

#84 shows the changes. 

Please let me know if that fixes your problem.

**Comment by wirthual** [comment]
Fixed by #84
